HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/Report/Report.cshtml

```diff
@@ -1,14 +1,22 @@
 @model HelpMyStreetFE.Models.Account.Report.ReportViewModel
 
 <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+
 <div>
-    <div class="chart-container" 
-         data-chart="@Model.Chart" 
-         data-group="@Model.GroupId" 
-         data-type="@Model.ChartType" 
-         data-datefrom="@Model.DateFrom" 
+    <div class="chart-container"
+         data-chart="@Model.Chart"
+         data-group="@Model.GroupId"
+         data-type="@Model.ChartType"
+         data-datefrom="@Model.DateFrom"
          data-dateto="@Model.DateTo">
-        <canvas class="chart"></canvas>
+        @if (Model.ChartType == HelpMyStreet.Utils.Enums.ChartTypes.DataTable)
```

Comment:
We could remove lines 12-19 here...

To achieve that, the JavaScript would need to add the appropriate element (either `<div>` or `<canvas>`) to the DOM (page) before inserting the content.